concent_api/concent_api/constants.py

```diff
@@ -34,3 +34,23 @@
         ],
     }),
 ])
+
+ERROR_CODE_GATEKEEPER_INVALID_REQUEST_BODY                      = 'gatekeeper.invalid_request.body'
+ERROR_CODE_GATEKEEPER_INVALID_REQUEST_HEADER_CONTENT_TYPE       = 'gatekeeper.invalid_request.header.content_type'
+ERROR_CODE_GATEKEEPER_INVALID_REQUEST_HEADER_AUTHORIZATION      = 'gatekeeper.invalid_request.header.authorization'
+ERROR_CODE_GATEKEEPER_INVALID_REQUEST_HEADER_CLIENT_PUBLIC_KEY  = 'gatekeeper.invalid_request.header.client_public_key'
+ERROR_CODE_GATEKEEPER_INVALID_REQUEST_MESSAGE                   = 'gatekeeper.invalid_request.message'
+ERROR_CODE_GATEKEEPER_INVALID_REQUEST_MESSAGE_FILES             = 'gatekeeper.invalid_request.message.files'
+ERROR_CODE_CORE_INVALID_REQUEST_HEADER_CONTENT_TYPE             = 'core.invalid_request.header.content_type'
+ERROR_CODE_CORE_INVALID_REQUEST_HEADER_CLIENT_PUBLIC_KEY        = 'core.invalid_request.header.client_public_key'
+ERROR_CODE_CORE_INVALID_REQUEST_HEADER_OTHER_PARTY_PUBLIC_KEY   = 'core.invalid_request.header.other_party_public_key'
+ERROR_CODE_CORE_INVALID_REQUEST_MESSAGE                         = 'core.invalid_request.message'
+ERROR_CODE_CORE_INVALID_REQUEST_MESSAGE_TASK_ID                 = 'core.invalid_request.message.task_id'
+ERROR_CODE_CORE_INVALID_REQUEST_MESSAGE_VALUE_NOT_INTEGER       = 'core.invalid_request.message.value_not_integer'
+ERROR_CODE_CORE_QUEUE_MESSAGE_ALREADY_PROCESSED                 = 'core.queue.message_already_processed'
+ERROR_CODE_CORE_QUEUE_COMMUNICATION_NOT_STARTED                 = 'core.queue.communication_not_started'
+ERROR_CODE_CORE_QUEUE_RESPONSE_ALREADY_SUBMITTED                = 'core.queue.response_already_submitted'
+ERROR_CODE_CORE_QUEUE_TIMEOUT                                   = 'core.queue.timeout'
+ERROR_CODE_CORE_MESSAGE_UNKNOWN                                 = 'core.message.unknown'
+ERROR_CODE_CORE_MESSAGE_UNEXPECTED                              = 'core.message.unexpected'
+ERROR_CODE_CORE_MESSAGE_UNABLE_TO_DESERIALIZE                   = 'core.message.unable_to_deserialize'
```

Comment:
I'm not sure if using app name as a prefix is a good idea. It's an implementation detail that the client should not care about. And this will be a problem when we start sharing more code between apps. We'll have to know which app is using the code which breaks encapsulation.

Reply:
:+1: